src/main/java/school/lemon/changerequest/java/introduction/hw1/Task9.java

```diff
@@ -2,7 +2,11 @@
 
 public class Task9 {
 
-    public static boolean isPowerOfThree(int n) {
-        return false;
+    public static boolean isPowerOfThree(int n)
+    {
+        double Rezult=Math.log(n)*(int)(Math.log(3))/((int)(Math.log(n))*(Math.log(3)));
```

Comment:
isPowerOfThree should return true for n=1 - FAILED
3^0 = 1, so 1 is of power of three.
Please fix it.